packages/core/package.json

```diff
@@ -10,12 +10,7 @@
         "serve:crypto": "f() { echo \"Serving...\" && sudo ./node_modules/.bin/webpack-dev-server --config \"./build/webpack.config.js\" --env.open $1 --env.IS_CRYPTO_APP=${IS_CRYPTO_APP:-false};}; f",
         "build": "f () { webpack --config \"./build/webpack.config.js\" --env.base $1 --env.IS_CRYPTO_APP=${IS_CRYPTO_APP:-false};}; f",
         "build:travis": "echo \"No build:travis specified\"",
-        "prettify": "prettier --write \"./{script,src,build}/**/*.{js,jsx,scss}\"",
-        "precommit": "lint-staged",
-        "test": "npm run test:stylelint && npm run test:eslint",
-        "test:temp": "mochapack -r babel-polyfill -r jsdom-global/register -r mock-local-storage --webpack-config \"./build/webpack.config-test.js\" \"src/**/__tests__/*.js\" --require ignore-styles",
-        "test:eslint": "eslint \"./src/**/*.?(js|jsx)\"",
-        "test:stylelint": "stylelint \"./src/sass/**/*.s?(a|c)ss\"",
+        "test": "echo \"Running mocha tests.\"",
```

Comment:
🤔

Reply:
test:temp was unused previously, and the latter 2 were moved to root. I've placed this as a placeholder for the next PR I'm working on to pick it up.